src/onelogin/saml2/response.py

```diff
@@ -128,14 +128,15 @@ def is_valid(self, request_data, request_id=None):
                     raise Exception('There is an EncryptedAttribute in the Response and this SP not support them')
 
                 # Checks destination
-                destination = self.document.get('Destination', '')
-                if destination:
-                    if not destination.startswith(current_url):
-                        # TODO: Review if following lines are required, since we can control the
-                        # request_data
-                        #  current_url_routed = OneLogin_Saml2_Utils.get_self_routed_url_no_query(request_data)
-                        #  if not destination.startswith(current_url_routed):
-                        raise Exception('The response was received at %s instead of %s' % (current_url, destination))
+                if security.get('checkDestination', True):
```

Comment:
Destination match should be relaxeable if domain is a correct match, this if for testing purposes.